### DISCUSSION TITLE
2024-04-22 (월)

### DISCUSSION THREAD

**Discussion by devwqc** [19-Takify/19-taskify#41] [회의록]
# 📣회의록
- 스크럼 & 역할 분담

# 🐾회고록
### Keep
- 팀 활동 참석을 잘한다.
- 문제가 발생한 부분을 서로 도와주려고 한다.
- 귀찮을 수 있는 일도 먼저 나서서 진행한다.
- 팀 컨벤션을 잘 지키려고 노력한다.
- 소통이 잘 된다.
- 팀원들 모두가 팀 활동에 적극적으로 참여한다.
- 프로젝트 진행 속도가 빠르다.
- 코드 리뷰를 통해 서로의 지식을 공유한다.
- pr을 올리면 꼼꼼하게 리뷰해준다.
- 프로젝트 진행이 원활하다.

### Problem
- 팀 미팅에 씻고 참여하는 팀원이 거의 없다.
- git 숙련도가 부족해서 이슈가 생기는 일이 있다.
- 처음 사용해보는 기술 스택들이 있어 어려움이 있다.
- 진행 속도가 빨라서 몇몇 분들이 할 일이 없다.
- 전체적으로는 빠르게 진행되고 있지만 인증 관련 부분이 계획보다 늦어지고 있다. (뜨끔...)
- 다들 잠은 자나 궁금하다. 너무 빠르다.
- git에 대한 공부가 더 필요하다.

### Try
- 코드, 파일 날려보면서 git 숙련도 쌓기.
- 지금처럼 잘 진행하면 좋겠다.
- 공식 문서를 활용해 실습해 보면서 익혀 나가야 할 것 같다.
- 어떤 것을 더 해야 하는지 잘 생각해보고 그래도 할 일이 없으면 추가 기능에 대해 더 생각해본다.
- 인증을 맡은 분들은 더 힘내서 열심히 한다.
- 나는 Git 공부가 더 필요하다.
- 프로젝트가 끝날 때까지 무사히 기능 완수 했으면 좋겠다.

### 직전 작업
- 회원가입 페이지 merge 되면 디자인 반영

### 할 일
- 유저 로그인 상태 유지

### 어려운 점
- 인증에 대한 부분이 감이 오질 않는다

### 직전 작업
- Modal 레이아웃 작업(모바일 미구현)

### 할 일
- Modal 레이아웃 작업(모바일 구현)

### 어려운 점
- git pull 하고 소스에 에러가 생김

### 직전 작업
- 내 대시보드 검색 기능 구현
- 대시보드 컬러 컴포넌트 분리

### 할 일
- 예비군

### 어려운 점
- 예비군이 어려움

### 직전 작업
- 대시보드 수정 페이지

### 할 일
- 대시보드 페이지 기술 고민

### 어려운 점
- 없음

### 직전 작업
- 회원가입 PR 피드백 최종 반영해서 머지
- axios interceptor & next middleware 테스트 중

### 할 일
- axios interceptor & next middleware 적용
- 회원가입 이메일 인증 시도

### 어려운 점
- 인증...

**Comment by devwqc** [comment]
## Keep
- 팀 활동 참석을 잘한다.
- 문제가 발생한 부분을 서로 도와주려고 한다.
- 귀찮을 수 있는 일도 먼저 나서서 진행한다.
- 팀 컨벤션을 잘 지키려고 노력한다.

## Problem
- git 숙련도가 부족해서 이슈가 생기는 일이 있다.

## Try
- 코드, 파일 날려보면서 git 숙련도 쌓기.
- 지금처럼 잘 진행하면 좋겠다.

**Comment by woo-29** [comment]
## Keep
- 소통이 잘 된다.

## Problem
- 처음 사용해보는 기술 스택들이 있어 어려움이 있다.

## Try
- 공식 문서를 활용해 실습해 보면서 익혀 나가야 할 것 같다.

**Comment by q45402sk** [comment]
### Keep
- 팀원들 모두가 팀 활동에 적극적으로 참여한다. 
- 프로젝트 진행 속도가 빠르다. 
- 코드 리뷰를 통해 서로의 지식을 공유한다. 

### Problem
- 진행 속도가 빨라서 몇몇 분들이 할 일이 없다. 
- 전체적으로는 빠르게 진행되고 있지만 인증 관련 부분이 계획보다 늦어지고 있다. (뜨끔...)

### Try
- 어떤 것을 더 해야 하는지 잘 생각해보고 그래도 할 일이 없으면 추가 기능에 대해 더 생각해본다. 
- 인증을 맡은 분들은 더 힘내서 열심히 한다.

**Comment by yb3143** [comment]
Keep
pr을 올리면 꼼꼼하게 리뷰해준다.

problem
다들 잠은 자나 궁금하다. 너무 빠르다.

Try
나는 Git 공부가 더 필요하다.

**Comment by skoo1100** [comment]
## Keep
- 팀원과의 소통이 잘 된다.
- 프로젝트 진행이 원활하다.

## Problem
- git에 대한 공부가 더 필요하다.

## Try
- 프로젝트가 끝날 때까지 무사히 기능 완수 했으면 좋겠다.
